android-lib/src/androidTest/java/com/kinvey/androidTest/store/DeltaCacheTest.java

```diff
@@ -568,12 +572,12 @@ public void testUpdateWithTwoStorageTypes() throws InterruptedException, IOExcep
         assertNull(saveCallback.getError());
         assertEquals(TEST_USERNAME + 100, saveCallback.getResult().getUsername());
 
-        CustomKinveyPullCallback<Person> pullCallback = testManager.pullCustom(store, client.query());
+        CustomKinveyPullCallback<Person> pullCallback = testManager.pullCustom(store, client.query().addSort("_kmd", AbstractQuery.SortOrder.ASC));
         assertNotNull(pullCallback);
         assertNotNull(pullCallback.getResult());
         assertNull(pullCallback.getError());
         assertEquals(TEN_ITEMS, pullCallback.getResult().getResult().size());
-        assertEquals(TEST_USERNAME + 100, pullCallback.getResult().getResult().get(0).getUsername());
+        assertEquals(TEST_USERNAME + 100, pullCallback.getResult().getResult().get(9).getUsername());
```

Comment:
Why did the index change here from **_0_** to **_9_**?

Reply:
I check the 9th (last item) because if we set sorting only by `_kmd` query will be sorted by `_kmd.lmt`' because `_kmd` object has `lmt` parameter before `ect`, example: `{"lmt":"2018-01-12T11:00:01.122Z","ect":"2018-01-12T11:00:01.122Z"}`.